dspy/teleprompt/teleprompt_optuna.py

```diff
@@ -52,6 +52,8 @@ def objective(self, trial):
             display_table=False,
             display_progress=True,
         )
+        # FIXME: this should have return_all_scores=False and then need only score,
```

Comment:
feel free to add that correction in this PR!

Reply:
Done!